src/userbase-js/src/auth.js

```diff
@@ -426,6 +428,7 @@ const signIn = async (params) => {
       case 'PasswordTooShort':
       case 'PasswordTooLong':
       case 'PasswordMustBeString':
+      case 'PasswordAttemptLimitExceeded':
```

Comment:
I think SignInAttemptLimitExceeded is better. What do you think?

Reply:
It's also protecting updateUser() in case someone is spamming incorrect current passwords via that endpoint. That's why I went with the general name.

Could make the names specific sign in + update user pretty easily. Do you have a good suggestion for update user?

Reply:
Makes sense. Let's leave it like this.